quantum/matrix.c

```diff
@@ -107,8 +111,11 @@ static bool read_cols_on_row(matrix_row_t current_matrix[], uint8_t current_row)
 
     // For each col...
     for (uint8_t col_index = 0; col_index < MATRIX_COLS; col_index++) {
-        // Select the col pin to read (active low)
-        uint8_t pin_state = readPin(col_pins[col_index]);
+        uint8_t pin_state = 0;
```

Comment:
If it's active low, should this not be 1?

Reply:
It might be, yeah. :(